src/main/java/org/terasology/blockNetwork/traversal/BreadthFirstTraversal.java

```diff
@@ -24,5 +24,13 @@
  * @param <V> Value passed from node visited into nodes that are outgoing from that node.
  */
 public interface BreadthFirstTraversal<T, U, V> {
+    /**
+     * Traverses the given node in the graph.
+     * This method is invoked for each node in a traversal as directed by the traversal definition and the traversal result.
```

Comment:
How does the "traversal result" direct the order of node traversal? If anything, i would have assumed the opposite - that the traversal dictates the result.

Reply:
The result influences the order through the predicate which specifies which node to traverse next and which not to.